[2024-04-29T12:24:23.220+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_metrics.publish.customer_transactions_run manual__2024-04-29T12:23:03.265543+00:00 [queued]>
[2024-04-29T12:24:23.226+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_metrics.publish.customer_transactions_run manual__2024-04-29T12:23:03.265543+00:00 [queued]>
[2024-04-29T12:24:23.227+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-29T12:24:23.237+0000] {taskinstance.py:2214} INFO - Executing <Task(DbtRunLocalOperator): publish.customer_transactions_run> on 2024-04-29 12:23:03.265543+00:00
[2024-04-29T12:24:23.242+0000] {standard_task_runner.py:60} INFO - Started process 7226 to run task
[2024-04-29T12:24:23.244+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'customer_metrics', 'publish.customer_transactions_run', 'manual__2024-04-29T12:23:03.265543+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/customer_metrics.py', '--cfg-path', '/tmp/tmp_rdqbxqi']
[2024-04-29T12:24:23.246+0000] {standard_task_runner.py:88} INFO - Job 39: Subtask publish.customer_transactions_run
[2024-04-29T12:24:23.275+0000] {task_command.py:423} INFO - Running <TaskInstance: customer_metrics.publish.customer_transactions_run manual__2024-04-29T12:23:03.265543+00:00 [running]> on host 925dffa277c3
[2024-04-29T12:24:23.328+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='customer_metrics' AIRFLOW_CTX_TASK_ID='publish.customer_transactions_run' AIRFLOW_CTX_EXECUTION_DATE='2024-04-29T12:23:03.265543+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-29T12:23:03.265543+00:00'
[2024-04-29T12:24:23.329+0000] {local.py:202} INFO - Cloning project to writable temp directory /tmp/tmph4e4jyd_ from /opt/airflow/include/dbt/fraud
[2024-04-29T12:24:23.332+0000] {config.py:253} INFO - Using user-supplied profiles.yml at /opt/airflow/include/dbt/fraud/profiles.yml
[2024-04-29T12:24:23.333+0000] {local.py:235} INFO - Trying to run the command:
 ['/home/airflow/.local/bin/dbt', 'run', '--models', 'customer_transactions', '--profiles-dir', '/opt/airflow/include/dbt/fraud', '--profile', 'fraud', '--target', 'dev']
From /tmp/tmph4e4jyd_
[2024-04-29T12:24:23.333+0000] {local.py:236} INFO - Using environment variables keys: dict_keys(['AIRFLOW_CTX_DAG_OWNER', 'AIRFLOW_CTX_DAG_ID', 'AIRFLOW_CTX_TASK_ID', 'AIRFLOW_CTX_EXECUTION_DATE', 'AIRFLOW_CTX_TRY_NUMBER', 'AIRFLOW_CTX_DAG_RUN_ID'])
[2024-04-29T12:24:23.334+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-04-29T12:24:23.334+0000] {subprocess.py:73} INFO - Running command: ['/home/airflow/.local/bin/dbt', 'run', '--models', 'customer_transactions', '--profiles-dir', '/opt/airflow/include/dbt/fraud', '--profile', 'fraud', '--target', 'dev']
[2024-04-29T12:24:23.341+0000] {subprocess.py:84} INFO - Command output:
[2024-04-29T12:24:25.286+0000] {subprocess.py:94} INFO - [0m12:24:25  Running with dbt=1.7.13
[2024-04-29T12:24:26.014+0000] {subprocess.py:94} INFO - [0m12:24:26  Registered adapter: snowflake=1.7.3
[2024-04-29T12:24:26.033+0000] {subprocess.py:94} INFO - [0m12:24:26  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-04-29T12:24:26.965+0000] {subprocess.py:94} INFO - [0m12:24:26  Found 3 models, 0 sources, 0 exposures, 0 metrics, 430 macros, 0 groups, 0 semantic models
[2024-04-29T12:24:26.966+0000] {subprocess.py:94} INFO - [0m12:24:26
[2024-04-29T12:24:28.131+0000] {subprocess.py:94} INFO - [0m12:24:28  Concurrency: 1 threads (target='dev')
[2024-04-29T12:24:28.132+0000] {subprocess.py:94} INFO - [0m12:24:28
[2024-04-29T12:24:28.138+0000] {subprocess.py:94} INFO - [0m12:24:28  1 of 1 START sql table model prod.customer_transactions ........................ [RUN]
[2024-04-29T12:24:29.230+0000] {subprocess.py:94} INFO - [0m12:24:29  1 of 1 OK created sql table model prod.customer_transactions ................... [[32mSUCCESS 1[0m in 1.09s]
[2024-04-29T12:24:29.233+0000] {subprocess.py:94} INFO - [0m12:24:29
[2024-04-29T12:24:29.234+0000] {subprocess.py:94} INFO - [0m12:24:29  Finished running 1 table model in 0 hours 0 minutes and 2.27 seconds (2.27s).
[2024-04-29T12:24:29.245+0000] {subprocess.py:94} INFO - [0m12:24:29
[2024-04-29T12:24:29.246+0000] {subprocess.py:94} INFO - [0m12:24:29  [32mCompleted successfully[0m
[2024-04-29T12:24:29.247+0000] {subprocess.py:94} INFO - [0m12:24:29
[2024-04-29T12:24:29.247+0000] {subprocess.py:94} INFO - [0m12:24:29  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-04-29T12:24:29.921+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-04-29T12:24:29.932+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v11.json is above dbt-ol supported version 7. This might cause errors.
[2024-04-29T12:24:29.940+0000] {local.py:252} INFO - Inlets: []
[2024-04-29T12:24:29.940+0000] {local.py:253} INFO - Outlets: [Dataset(uri='snowflake://grb43409.us-west-1.aws/AIRBYTE_DATABASE.prod.customer_transactions', extra=None)]
[2024-04-29T12:24:29.941+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-29T12:24:29.963+0000] {dag.py:3834} INFO - Setting next_dagrun for customer_metrics to 2024-04-29 00:00:00+00:00, run_after=2024-04-30 00:00:00+00:00
[2024-04-29T12:24:29.990+0000] {local.py:371} INFO - [0m12:24:29  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-04-29T12:24:29.995+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py:1469 RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
[2024-04-29T12:24:30.016+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=customer_metrics, task_id=publish.customer_transactions_run, execution_date=20240429T122303, start_date=20240429T122423, end_date=20240429T122430
[2024-04-29T12:24:30.063+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-29T12:24:30.077+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
